lib/documents.js

```diff
@@ -5,13 +5,14 @@ import * as bedrock from '@bedrock/core';
 import * as database from '@bedrock/mongodb';
 import * as entities from './entities.js';
 import canonicalize from 'canonicalize';
+import {Cipher} from '@digitalbazaar/minimal-cipher';
 import {createKeyResolver} from './keyResolver.js';
-import {createRequire} from 'node:module';
+import {IdGenerator} from 'bnid';
 import {tokenizers} from '@bedrock/tokenizer';
+
+import {createRequire} from 'node:module';
 const require = createRequire(import.meta.url);
 const assert = require('assert-plus');
```

Comment:
@davidlehn ... do we still need to be doing this `require` stuff with `assert-plus`?

Reply:
No, in most cases we can just use import with CJS modules.  It will do conversion internally.  I think there might be some edge cases, but they will be obvious failures.  This style with assert-plus was left in in a few places, but yeah, I think it can be removed.